lms/templates/dashboard.html

```diff
@@ -62,9 +62,10 @@
 </%block>
 
 <div class="dashboard-notifications" tabindex="-1">
-    %if message:
+
+    %if account_activation_message and not display_account_activation_message_on_sidebar:
```

Comment:
As above. We can simplify this substantially by having two separate variable names; one for the `sidebar activation message` and one for the `banner activation message`.